modules/contentbox/modules/contentbox-admin/resources/assets/js/editors/autosave.js

```diff
@@ -6,6 +6,23 @@
  * @param  {object} options  The override options such as: storeMax, timeout
  * @return {object}          Returns the auto save closure
  */
+import { decompressFromUTF16, compressToUTF16 } from "lz-string";
+import moment from "moment";
+
+window.resetAutoSave = function(){
+	if ( !Modernizr.localstorage ) return;
+	var saveStoreKey 	= "autosave_" + window.location;
```

Comment:
since the `autosave` prefix is now used in two locations, can you just create a constant on the class itself to avoid duplication and statically bind it to the compiler:

```
const KEY_PREFIX = "autosave_"
```